src/components/Controls/ToggleAudioButton/ToggleAudioButton.tsx

```diff
@@ -20,11 +20,7 @@ export default function ToggleAudioButton(props: { disabled?: boolean }) {
   const classes = useStyles();
   let [isAudioEnabled, toggleAudioEnabled] = useLocalAudioToggle();
   return (
-    <Tooltip
-      title={isAudioEnabled ? 'Mute Audio' : 'Unmute Audio'}
-      placement="top"
-      PopperProps={{ disablePortal: true }}
-    >
+    <Tooltip title={isAudioEnabled ? 'Audio off' : 'Audio on'} placement="top" PopperProps={{ disablePortal: true }}>
```

Comment:
But the customers ask from us to change it, Shani said me that "Please change the wording to "video on" and "video off" instead of mute/unmute video."

Reply:
@talr When you say layout, you mean the line breaks on the Tooltip component's props?
I can see some code that uses multi-line and some using inline props,
If there's no style guide we can use both styles, for short rows it can be inline and once it becomes longer than a typical screen, we can break it into multiple lines.

Reply:
I break it into multiple lines